src/game/default/g_entity.c

```diff
@@ -215,6 +280,7 @@ static const g_field_t fields[] = {
 	{ "gameplay", SOFS(gameplay), F_STRING, FFL_SPAWN_TEMP },
 	{ "hook", SOFS(hook), F_STRING, FFL_SPAWN_TEMP },
 	{ "teams", SOFS(teams), F_STRING, FFL_SPAWN_TEMP },
+	{ "num_teams", SOFS(num_teams), F_STRING, FFL_SPAWN_TEMP },
```

Comment:
Pretty sure I did fix the map list parser to support it. If the map has either 3 flags or 3 team spawns, it'll just work automatically anyways, but yeah it does pick it up (g_map_list.c line 128)